plenoptic/simulate/canonical_computations/steerable_pyramid_freq.py

```diff
@@ -79,7 +80,7 @@ class Steerable_Pyramid_Freq(nn.Module):
     """
 
     def __init__(self, image_shape, height='auto', order=3, twidth=1, is_complex=False,
-                  downsample=True, tight_frame=True):
+                  downsample=True,  tight_frame=False):
```

Comment:
I went back and forth on this.. the norm argument is about the actual normalization on the fft whereas the tight_frame has the sqrt(2) stuff inside the code. 
Like I guess since we don't provide an option to set the fft norm arguments directly I could just call this norm, but I didn't want to confuse it with the norm in numpy or pytorch which is a string ("ortho" , "backward", "forward")

Reply:
agreed with Kate's point -- would be nice to have some details on when someone would want to set this true or false. (might not belong here, could be in the notebook)

Reply:
yeah I think when we do a second pass on notebooks we can include that in the notebook because it's really dependent on the problem and usage and so it's not so easy to do just in the docstring for the class.